content/intro-to-storybook/ember/en/simple-component.md

```diff
@@ -0,0 +1,223 @@
+---
+title: 'Build a simple component'
+tocTitle: 'Simple component'
+description: 'Build a simple component in isolation'
+---
+
+We’ll build our UI following a [Component-Driven Development](https://blog.hichroma.com/component-driven-development-ce1109d56c8e) (CDD) methodology. It’s a process that builds UIs from the “bottom up” starting with components and ending with screens. CDD helps you scale the amount of complexity you’re faced with as you build out the UI.
+
+## Task
+
+![Task component in three states](/intro-to-storybook/task-states-learnstorybook.png)
+
+`Task` is the core component in our app. Each task displays slightly differently depending on exactly what state it’s in. We display a checked (or unchecked) checkbox, some information about the task, and a “pin” button, allowing us to move tasks up and down the list. Putting this together, we’ll need these props:
+
+- `title` – a string describing the task
+- `state` - which list is the task currently in and is it checked off?
+
+As we start to build `Task`, we first write our test states that correspond to the different types of tasks sketched above. Then we use Storybook to build the component in isolation using mocked data. We’ll “visual test” the component’s appearance given each state as we go.
+
+This process is similar to [Test-driven development](https://en.wikipedia.org/wiki/Test-driven_development) (TDD) that we can call “[Visual TDD](https://blog.hichroma.com/visual-test-driven-development-aec1c98bed87)”.
+
+## Get setup
+
+First, let’s create the necessary files for our task component and its accompanying story file: `app/components/Task.hbs` and `app/components/Task.stories.js`.
+
+We’ll begin with a basic implementation of the `Task`, simply taking in the attributes we know we’ll need and the two actions you can take on a task (to move it between lists):
+
+```handlebars
+
+{{!--  app/components/Task.hbs --}}
+
+<div class="list-item">
+  <input type="text" value={{@task.title}} readonly={{true}}/>
+</div>
+```
+
+Above, we render straightforward markup for `Task` based on the existing HTML structure of the Todos app.
+
+Below we build out Task’s three test states in the story file:
+
+```javascript
+//  app/components/Task.stories.js
+import { hbs } from 'ember-cli-htmlbars';
+import { action } from '@storybook/addon-actions';
+
+export default {
+  title: 'Task',
+  component: 'Task',
+  excludeStories: /.*Data$/,
+};
+
+export const taskData = {
+  id: '1',
+  title: 'Test Task',
+  state: 'TASK_INBOX',
+  updatedAt: new Date(2018, 0, 1, 9, 0),
+};
+
+export const actionsData = {
+  onPinTask: action('onPinTask'),
+  onArchiveTask: action('onArchiveTask'),
```

Comment:
```suggestion
  pinTask: action('pinTask'),
  archiveTask: action('archiveTask'),
```

Reply:
@gossi this was introduced to have consistency across the whole documentation. Calling it as it is or what you have is rather inconsequential. Sorry but i'm marking this as resolved and leaving it as is.